src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const valueOfType = action['type'];
```

Comment:
You can use a desctructurization here for action
```suggestion
  for (const action of actions) {
    const {} = action;
```